docs/source/standard/auction.rst

```diff
@@ -161,7 +159,7 @@ Schema
   Auction step (increment). Originates from `lots.auctions.minimalStep`.
 
 :awards:
-  List of :ref:`award` objects
+  Array of :ref:`award` objects
 
   All qualifications (disqualifications and awards).
 
```

Comment:
These should be added

**:minNumberOfQualifiedBids:** 
integer, auto-generated, read-only

Number of submitted bids for the process to become successful. The default value is 1.

**:submissionMethod:**
string, auto-generated, read-only

The given value is _electronicAuction._

**:awardCriteria:**
string, auto-generated, read-only

The given value is _highestCost._

**:owner:**
string, auto-generated, read-only

The entity whom the procedure has been created by. 

**: date:**
date, auto-generated, read-only

The date of the procedure creation/undoing.